---
int32 hintID1
int32 hintID2
